### PR TITLE
Support Mocha require option

### PR DESCRIPTION
**Pull request by silvenon**
Not sure if I did it correctly, but it seems to work fine. Do you also want tests for that?

### CODE REVIEW

**Comment by silvenon** [comment]
Hm, not sure what to do about the tests failing…

File: package.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
     "deepmerge": "^0.2.7"
   },
   "devDependencies": {
-    "gulp": "^3.8.11"
+    "gulp": "^3.8.11",
+    "resolve-from": "^1.0.0"
```
Comment (christian-bromann) [inline package.json]: this belongs into the dependency section

Comment (silvenon) [inline package.json]: Fixed.

**Comment by christian-bromann** [comment]
sorry for the delay, tests were failing because you didn't have the your sauce creds in the travis matrix, let's see how works out in master

**Comment by christian-bromann** [comment]
Thanks :+1:

**Pull request merged**